test/regress/cli/CMakeLists.txt

```diff
@@ -464,6 +464,120 @@ set(regress_0_tests
   regress0/bv/fuzz40.delta01.smtv1.smt2
   regress0/bv/fuzz40.smtv1.smt2
   regress0/bv/fuzz41.smtv1.smt2
+  # Added to fill holes in regression tests
+  regress0/bv/holes/concat-flatten.smt2
```

Comment:
These need to be alphabetized. Also there should not be spaces between the benchmarks or comments.

Reply:
fixed